Bottom: 0c0ed37df04e3b3982c1c6dbd99241005b174938
Top:    d3e59cede44c8b8296ca26dc2c5c941d3aa3b3ae
Author: Andrew Cooper <andrew.cooper@novatechweb.com>
Date:   2018-06-04 17:31:43 -0500

Fix credential copy to only run on daedalus

Credentials for all hosts end up on daedalus, so limit that play.


---
diff --git a/ansible-playbook/site.yml b/ansible-playbook/site.yml
index db9517d2..9827d861 100644
--- a/ansible-playbook/site.yml
+++ b/ansible-playbook/site.yml
@@ -117,18 +117,11 @@
     assemble_scripts: true
 
 - name: Cleanup tasks
-  hosts: all
+  hosts: daedalus
   tasks:
   - name: Copy credentials to the /etc dir
     copy:
-      src: ./credentials/{{ inventory_hostname }}
-      dest: /etc/ansible-credentials/
-      owner: root
-      group: root
-      mode: 'u=rw,g=,o='
-  - name: Copy build system credentials to /etc dir
-    copy:
-      src: ./credentials/buildsystem.novatech-llc.com
+      src: "{{ playbook_dir }}/credentials/"
       dest: /etc/ansible-credentials/
       owner: root
       group: root
